### PR TITLE
Receiving (test func) and sending transactions wrappers

### PR DESCRIPTION
**Pull request by Jasonvdb**
- Send TX test
- Update Libs
- Fixing log path.
- Testing for pendingIncomingBalance.
- Only run unit tests on travis for now

### CODE REVIEW

File: MobileWallet/TariLib/Wrappers/Transactions/CompletedTransaction.swift [hunk 1]
```diff
@@ -40,13 +40,42 @@
 
 import Foundation
 
+enum CompletedTransactionStatus: Error {
+    case transactionNullError
+    case completed
+    case broadcast
+    case mined
+    case unknown
```
Comment (StriderDM) [inline MobileWallet/TariLib/Wrappers/Transactions/CompletedTransaction.swift]: There isn't a status returned for this

File: MobileWallet/TariLib/Wrappers/Transactions/CompletedTransaction.swift [hunk 2]
```diff
@@ -40,13 +40,42 @@
 
 import Foundation
 
+enum CompletedTransactionStatus: Error {
```
Comment (StriderDM) [inline MobileWallet/TariLib/Wrappers/Transactions/CompletedTransaction.swift]:
```suggestion+7
enum CompletedTransactionStatus: Error {
    case transactionNullError
    case completed
    case broadcast
    case mined
}
```

Comment (StriderDM) [inline MobileWallet/TariLib/Wrappers/Transactions/CompletedTransaction.swift]:
```suggestion:+7
enum CompletedTransactionStatus: Error {
    case transactionNullError
    case completed
    case broadcast
    case mined
}
```

File: MobileWallet/TariLib/Wrappers/Transactions/CompletedTransaction.swift [hunk 3]
```diff
@@ -40,13 +40,42 @@
...
 
     var pointer: OpaquePointer {
         return ptr
     }
 
+    var id: UInt64 {
+        return completed_transaction_get_transaction_id(ptr)
+    }
+
+    var status: CompletedTransactionStatus {
```
Comment (StriderDM) [inline MobileWallet/TariLib/Wrappers/Transactions/CompletedTransaction.swift]:
```suggestion:+15
    var status: CompletedTransactionStatus {
        let status: Int32 = completed_transaction_get_status(ptr)
        switch status {
        case 0:
            return .completed
        case 1:
            return .broadcast
        case 2:
            return .mined
        default:
            return .transactionNullError
        }
    }
```

File: MobileWallet/TariLib/Wrappers/Wallet.swift [hunk 4]
```diff
@@ -47,6 +47,8 @@ enum WalletErrors: Error {
     case generateTestData
     case generateTestReceiveTransaction
     case testTransactionBroadcast
+    case testTransactionMined
+    case sending
```
Comment (StriderDM) [inline MobileWallet/TariLib/Wrappers/Wallet.swift]: ```sendingTransaction```

File: MobileWalletTests/TariLibWrapperTests.swift [hunk 5]
```diff
@@ -89,18 +94,22 @@ class TariLibWrapperTests: XCTestCase {
...
+        
+        print(loggingFilePath)
+        print(databasePath)
 
         let comsConfig = CommsConfig(
             privateKey: PrivateKey(hex: privateKeyHex),
             databasePath: databasePath,
             databaseName: dbName,
-            address: "0.0.0.0:80"
+            controlAddress: "0.0.0.0:80",
```
Comment (StriderDM) [inline MobileWalletTests/TariLibWrapperTests.swift]:
```suggestion:+1
            controlAddress: "127.0.0.1:80",
            listenerAddress: "0.0.0.0:80"
```

File: MobileWallet/TariLib/Wrappers/Wallet.swift (hunk 4 above)
Comment (Jasonvdb) [reply MobileWallet/TariLib/Wrappers/Wallet.swift]: done

**Pull request merged**